app/_src/guides/upgrading-transparent-proxy.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Upgrading Transparent Proxy
+content_type: tutorial
+---
+
+{% capture docs %}/docs/{{ page.release }}{% endcapture %}
+{% assign Kuma = site.mesh_product_name %}
+
+{% capture Warning %}{% if page.edition and page.edition != "kuma" %}**Warning:** {% endif %}{% endcapture %}
+{% capture Important %}{% if page.edition and page.edition != "kuma" %}**Important:** {% endif %}{% endcapture %}
+{% capture brbr %}{% if page.edition and page.edition != "kuma" %}<br /><br />{% endif %}{% endcapture %}
+
+The core `iptables` rules applied by {{ Kuma }}'s transparent proxy rarely change, but occasionally new features may require updates. To upgrade the transparent proxy on Universal environments, follow these steps:
+
+### Step 1: Cleanup existing iptables rules{% if_version gte:2.9.x inline:true %} (conditional){% endif_version %}
+
+{% if_version gte:2.9.x %}
```

Comment:
Is it possible to invert it? To have a warning only on docs older than 2.9, and on 2.9 and newer have it clean?

Reply:
but this warning has only sense for 2.9+

Reply:
ah ok, I misread it, btw should be `{% if_version gte:2.10.x %}` as you don't need to see the warning on 2.9

Reply:
I don't think so, because there might be the rare situation where you want to upgrade from 2.9.x to different version of 2.9 for example